core/include/lifo.h

```diff
@@ -70,7 +101,24 @@ void lifo_insert(int *array, int i);
  * @return          -1, if the lifo is empty.
  * @return          the least recently inserted element, otherwise.
  */
-int lifo_get(int *array);
+static inline int lifo_get(int *array)
+{
+    DEBUG("lifo_get\n");
```

Comment:
Missed one ;-).

Reply:
Thank you :) 
Fixed both.
(I even loaded my old HEAD just to check if this was an error in the squashing, no it wasn't i was there before)